app/ledger.js

```diff
@@ -470,7 +470,8 @@ var updatePublisherInfo = () => {
     var entries = []
 
     underscore.keys(publishers[publisher]).forEach((location) => {
```

Comment:
in this case, no... because in the function i need access to both the key and value, not just the value...